source/infrastructure/free-radius/manual-freeradius-test.md.erb

```diff
@@ -0,0 +1,82 @@
+---
+title: Test FreeRADIUS on the ECS task level
+weight: 70
+last_reviewed_on: "2023-08-01"
+review_in: 6 months
+---
+
+If you need to test the new FreeRADIUS ECS tasks manually (e.g. when the new Environmnet is created and Smoke tests are not deployed yet) you can use the steps listed below.
+
+## Prerequsites
+
+We will be using the ECS Exec so please ensure your machine is configure in line with the Prerequsites listed [here](https://govwifi-dev-docs.cloudapps.digital/infrastructure/access-ecs-tasks.html) and you are connected to the VPN.
```

Comment:
Links that just say 'here' are not very accessible, if I tab through a document all I hear is 'here' could we add some context to the links, like 'you can check your machines prerequisites here'
```suggestion
We will be using the ECS Exec so please ensure your machine is configure in line with the prerequisites listed, [you can check your machines prerequisites here](https://govwifi-dev-docs.cloudapps.digital/infrastructure/access-ecs-tasks.html) and you are connected to the VPN.
```